ext/date/php_date.c

```diff
@@ -2816,7 +2822,28 @@ PHP_METHOD(DateTimeImmutable, __construct)
 }
 /* }}} */
 
-/* {{{ proto DateTimeImmutable::createFromMutable(DateTimeZone object)
+/* {{{ proto DateTime::createFromImmutable(DateTimeImmutable object)
+   Creates new DateTime object from an existing immutable DateTimeImmutable object.
```

Comment:
Would it not make more sense to group this with the other DateTime methods, instead of just before the DateTimeImmutable/createFromMutable method?